src/test/java/marquez/client/MarquezClientTest.java

```diff
@@ -321,6 +340,36 @@ public void testGetDbTable() throws Exception {
     assertThat((DbTable) dataset).isEqualTo(DB_TABLE);
   }
 
+  @Test
+  public void testModifiedDbTable() throws Exception {
+    final String pathTemplate = "/namespaces/%s/datasets/%s";
+    final String path = buildPathFor(pathTemplate, DEFAULT_NAMESPACE_NAME, DB_TABLE_NAME);
+    final URL url = buildUrlFor(path);
+    when(http.url(pathTemplate, DEFAULT_NAMESPACE_NAME, DB_TABLE_NAME)).thenReturn(url);
+
+    final String dbTableAsJson = JsonGenerator.newJsonFor(DB_TABLE);
+    when(http.get(url)).thenReturn(dbTableAsJson);
+
+    final DbTable dataset = (DbTable) client.getDataset(DB_TABLE_NAME);
+
+    final DbTableMeta modifiedMeta =
+        DbTableMeta.builder()
+            .physicalName(dataset.getPhysicalName())
+            .sourceName(dataset.getSourceName())
+            .description(dataset.getDescription().get())
+            .runId(RUN_ID)
+            .build();
+
+    final String modifiedMetaAsJson = JsonGenerator.newJsonFor(modifiedMeta);
+    final String modifiedDbTableAsJson = JsonGenerator.newJsonFor(DB_TABLE_MODIFIED);
+    when(http.put(url, modifiedMetaAsJson)).thenReturn(modifiedDbTableAsJson);
+
+    final Dataset modifiedDataset = client.createDataset(DB_TABLE_NAME, modifiedMeta);
+    assertThat(modifiedDataset).isInstanceOf(DbTable.class);
+    assertThat((DbTable) modifiedDataset).isEqualTo(DB_TABLE_MODIFIED);
+    assertThat(modifiedDataset.getLastModified()).isNotNull();
```

Comment:
Given that you check for object equality:

```java
assertThat((DbTable) modifiedDataset).isEqualTo(DB_TABLE_MODIFIED);
```

I think we can remove the assertion:

```java
assertThat(modifiedDataset.getLastModified()).isNotNull();
```

or assert that `Dataset.lastModified` is after a timestamp `beforeModified`. For example:

```java
final Instant beforeModified = Instant.now();
// test logic
assertThat(modifiedDataset.getLastModified()).isAfter(beforeModified);

```

Reply:
Done